src/component/Timer/index.jsx

```diff
@@ -14,37 +14,52 @@ const Timer = () => {
   const [isRunning, setIsRunning] = useState(false);
   const [currentPeriod, setCurrentPeriod] = useState(INITIAL_PERIOD);
   const tickTimeoutId = useRef(0);
-  const counter = useRef(0);
+  const [counter, setCounter] = useState(1);
+
+  const timeOver = () => {
```

Comment:
Remember that function names should start with a verb. `time` is not a verb. What do you think of changing the name to something like `endCurrentPeriod`, `moveToNextPeriod`, `startNextPeriod` or even `endCurrentAndStartNextPeriod`? I think all of these are good options, which one would you choose? You can choose something different too if you can come up with a better name, just make sure that it starts with a verb and it's clear what it is doing.